webplatformcompat/viewsets.py

```diff
@@ -223,7 +246,7 @@ def include_child_pages(self):
         request = getattr(self, 'request')
         child_pages = request and request.query_params.get('child_pages')
```

Comment:
Wrote a lot more in the docstring in 8a04ff0ef38d7fa20d8c8e489ac57d3cb1a65b6b, plus some code changes.